lms/static/js/views/fields.js

```diff
@@ -116,8 +141,8 @@
             },
 
             showSuccessMessage: function () {
-                var successMessage = this.getMessage('success');
-                this.showNotificationMessage(successMessage);
+                var successMessage = this.getMessage('success').toString();
```

Comment:
+1, why are we calling `getMessage('success')` twice in succession here? Either remove this line, or use the variable in the following line.